backend/server.js

```diff
@@ -0,0 +1,224 @@
+const express = require("express");
+const cors = require("cors");
+const multer = require("multer");
+
+const upload = multer();
+
+const cookieParser = require("cookie-parser");
+const app = express();
+
+const MongoClient = require("mongodb").MongoClient;
+
+const passwordCheck = require(__dirname + "/utilities/passwordCheck.js");
+const generateRandomString = require(__dirname +
+  "/utilities/generateRandomString.js");
+const passwordHash = require("password-hash");
+
+// app.listen(4000, () => {
+//    // LOCAL SERVER
+//    console.log("Running on port 4000");
+// });
+//_____________________MOGODB Setup___________________
+
+let dataBaseUrl =
+  "mongodb+srv://admin:WhqtQsNqxt@cluster0-58fms.mongodb.net/test?retryWrites=true&w=majority";
+
+let UsersDB;
+let Collection_LoginInfo;
+let Collection_Sessions;
```

Comment:
It's a bit weird to have "nested" Capitalized vars like this. I think I would do:

```
const Collection = {
  LoginInfo: null,
  Sessions: null,
}
```

Then you can access them with `Collection.LoginInfo`.